### ISSUE TITLE
Zenith: panic in TUI

### ISSUE THREAD

**Issue by shykes** [dagger/dagger#5891]
## Dagger version

Dev version, built from main in the last 48 hours. (I don't remember from which commit, and there's no way to know from the binary).

## Environment

```console
$ uname -msv
Darwin Darwin Kernel Version 21.6.0: Wed Aug 10 14:28:23 PDT 2022; root:xnu-8020.141.5~2/RELEASE_ARM64_T6000 arm64
```

## What I did

In a development version of `github.com/shykes/daggerverse/dagger`, with uncommitted changes:

```console
dagger call
```

## What I expect

Successful execution, a list of available functions in the module

## What I got instead

I had to take a screenshot, because the crash leaves my terminal broken, and selection (for selecting and copy-pasting) get dumped as text instead of executed.

<img width="594" alt="Screen Shot 2023-10-13 at 10 56 31 AM" src="https://github.com/dagger/dagger/assets/29565/12a79efa-91a7-4df4-ac88-6aa97fc16917">

**Comment by jedevc** [comment]
This should be fixed by https://github.com/dagger/dagger/pull/5882.

**Comment by helderco** [comment]
Yes, that part of the code doesn't even exist in #5882 anymore and the codegen bug that probably affected this was fixed in #5881.
